src/SFA.DAS.EmployerFinance.Web/DependencyResolution/DefaultRegistry.cs

```diff
@@ -0,0 +1,19 @@
+using StructureMap;
+using SFA.DAS.EmployerFinance.Web.Urls;
+
+namespace SFA.DAS.EmployerFinance.Web.DependencyResolution
+{
+    public class DefaultRegistry : Registry
+    {
+        public DefaultRegistry()
+        {
+            For<IEmployerUrls>().Use<EmployerUrls>();
+
+//            Scan(s =>
```

Comment:
Could the commented out code be removed?

Reply:
Removed.